change_log/next/FE-2159-link-component.yml

```diff
@@ -0,0 +1 @@
+Improvement: "Refactor component to be more automated. Fixed known bugs. (Component: Link)"
```

Comment:
this change log entry does not correspond to changes made in this PR. "Fixed known bugs" doesn't say much.

Reply:
You are right. Updated